src/chains.ts

```diff
@@ -92,6 +94,19 @@ const chains: { [key in ChainId]: Chain } = {
     isValidTransactionHash: (hash: string) => isValidNearTx(hash),
     formatTransactionHash: (hash: string) => hash
   },
+  [ChainId.Solana]: {
+    name: 'Solana',
+    code: 'SOL',
+    nativeAsset: 'SOL',
+    fees: {
+      unit: 'SOL'
+    },
+    safeConfirmations: 0,
```

Comment:
What does it mean that safeconfirmations is 0? does solana not have a concept of confirmations?